<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyDialog</class>
 <widget class="QDialog" name="keyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>193</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>unlock and mount a luks volume</string>
  </property>
  <widget class="QPushButton" name="pbOpen">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditKey">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbkeyOption">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenMountPoint">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMountPoint">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxOpenReadOnly">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount In &amp;Read Only Mode</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShareMountPoint">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Share Mount Point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOptions">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>O&amp;ptions</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbKeyType">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="veraCryptWarning">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditKey</tabstop>
  <tabstop>lineEditMountPoint</tabstop>
  <tabstop>pbOpenMountPoint</tabstop>
  <tabstop>cbKeyType</tabstop>
  <tabstop>pbkeyOption</tabstop>
  <tabstop>checkBoxOpenReadOnly</tabstop>
  <tabstop>checkBoxShareMountPoint</tabstop>
  <tabstop>pbOptions</tabstop>
  <tabstop>pbOpen</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
